nipype/interfaces/ants/segmentation.py

```diff
@@ -692,6 +693,35 @@ class BrainExtraction(ANTSCommand):
     output_spec = BrainExtractionOutputSpec
     _cmd = 'antsBrainExtraction.sh'
 
+    def _run_interface(self, runtime, correct_return_codes=(0,)):
+        # antsBrainExtraction.sh requires ANTSPATH to be defined
+        out_environ = self._get_environ()
+        if out_environ.get('ANTSPATH') is None:
+            runtime.environ.update(out_environ)
+            executable_name = self.cmd.split()[0]
+            exist_val, cmd_path = _exists_in_path(executable_name, runtime.environ)
+            if not exist_val:
+                raise IOError("command '%s' could not be found on host %s" %
+                              (self.cmd.split()[0], runtime.hostname))
+
+            # Set the environment variable if found
+            runtime.environ.update({'ANTSPATH': os.path.dirname(cmd_path)})
```

Comment:
i believe `ANTSPATH` requires a trailing `/`. could you confirm this and make sure we adjust for it?